### PR TITLE
updated version of stack linked list

### PR DESCRIPTION
**Pull request by anxkhn**
updated version of stack linked list with proper formatting and fixed print statements.

#### Checklist

- [x] Added description of change
- [x] Relevant documentation/comments is changed or added
- [x] PR title follows semantic [commit guidelines](https://github.com/TheAlgorithms/C-Plus-Plus/blob/master/CONTRIBUTING.md#Commit-Guidelines)
- [x] Search previous suggestions before making a new one, as yours may be a duplicate.
- [x] I acknowledge that all my contributions will be made under the project's license.

Notes:

### CODE REVIEW

**Review by mishraabhinn — APPROVED** [review]

File: data_structures/stack_using_linked_list.cpp [hunk 1]
```diff
@@ -1,64 +1,76 @@
 #include <iostream>
```
Comment (realstealthninja) [inline data_structures/stack_using_linked_list.cpp:1]:
```suggestion
#include <iostream> /// for I/O operations 
```

File: data_structures/stack_using_linked_list.cpp [hunk 2]
```diff
@@ -1,64 +1,76 @@
...
+            break;
+        case 2:
+            pop();
+            break;
+        case 3:
+            show();
+            break;
+        default:
+            std::cout << "Invalid option!\n";
+            break;
```
Comment (realstealthninja) [inline data_structures/stack_using_linked_list.cpp:73]: please use tests

File: data_structures/stack_using_linked_list.cpp [hunk 3]
```diff
@@ -1,64 +1,76 @@
...
+void show()
+{
     node *t = top_var;
-    while (t != nullptr) {
+    while (t != nullptr)
+    {
         std::cout << t->val << "\n";
         t = t->next;
     }
 }
```
Comment (realstealthninja) [inline data_structures/stack_using_linked_list.cpp:44]:
```suggestion
/**
 * @brief Main function
 * @returns 0 on exit
 */
```

**Review by realstealthninja — CHANGES_REQUESTED** [review]

**Comment by realstealthninja** [comment]
closing due to no response from author, please comment if you'd like this to be re opened

**Pull request closed without merging**